app/views/coronavirus_landing_page/components/hub/_guidance_section.html.erb

```diff
@@ -9,7 +9,7 @@
             font_size: 19,
             margin_bottom: 5
           } %>
-          <ul class="covid__list govuk-!-margin-bottom-8">
+          <ul class="govuk-list covid__list govuk-!-margin-bottom-8">
```

Comment:
Could be swapped at some point to the [list gem](https://components.publishing.service.gov.uk/component-guide/list)

Reply:
Unfortunately it can't in this case - the `covid_list` does some extra things and the list component (along with all the other components) doesn't allow custom classes to be added.